### PR TITLE
Make i3wm internally consistent with itself

### PR DESCRIPTION
**Pull request by EdJoJob**
- Stop vim-modeline attempted interpretation
- Remove use of literal `ten` when there is a `<number_small>` capture
- Simplify readability of multiple window close commands in i3wm
- i3wm window movements separated
- i3wm match window flipper to port flipper

### CODE REVIEW

**Review by rntz — APPROVED** [review]
Lgtm but I don't use i3wm, I think fidget does though.

File: apps/i3wm/i3wm.talon [hunk 1]
```diff
@@ -61,13 +61,13 @@ vertical (shell|terminal):
...
-(shuffle|move (win|window) [to] port) <number_small>:  user.system_command("i3-msg move container to workspace number {number_small}")
-(shuffle|move (win|window) [to] port ten): user.system_command("i3-msg move container to workspace number 10")
-(shuffle|move (win|window) [to] last port): user.system_command("i3-msg move container to workspace back_and_forth")
-(shuffle|move (win|window) left): user.system_command("i3-msg move left")
-(shuffle|move (win|window) right): user.system_command("i3-msg move right")
-(shuffle|move (win|window) up): user.system_command("i3-msg move up")
-(shuffle|move (win|window) down): user.system_command("i3-msg move down")
+(shuffle|move) [(win|window)] [to] port <number_small>:  user.system_command("i3-msg move container to workspace number {number_small}")
+(shuffle|move) [(win|window)] [to] last port: user.system_command("i3-msg move container to workspace back_and_forth")
+(shuffle|move) flipper: user.system_command("i3-msg move container to workspace back_and_forth")
```
Comment (rntz) [inline apps/i3wm/i3wm.talon:66]: is this distinct from `port flip | flipper`?

File: apps/i3wm/i3wm.talon [hunk 2]
```diff
@@ -61,13 +61,13 @@ vertical (shell|terminal):
...
 # XXX - just replace with shuffle eventually?
 # XXX - like also need to match the generic talon commands
-(shuffle|move (win|window) [to] port) <number_small>:  user.system_command("i3-msg move container to workspace number {number_small}")
-(shuffle|move (win|window) [to] port ten): user.system_command("i3-msg move container to workspace number 10")
-(shuffle|move (win|window) [to] last port): user.system_command("i3-msg move container to workspace back_and_forth")
-(shuffle|move (win|window) left): user.system_command("i3-msg move left")
-(shuffle|move (win|window) right): user.system_command("i3-msg move right")
-(shuffle|move (win|window) up): user.system_command("i3-msg move up")
-(shuffle|move (win|window) down): user.system_command("i3-msg move down")
+(shuffle|move) [(win|window)] [to] port <number_small>:  user.system_command("i3-msg move container to workspace number {number_small}")
```
Comment (rntz) [inline apps/i3wm/i3wm.talon:64]: This is a different command grammar. The previous one accepts eg. `shuffle five`, but this one requires you to say `shuffle port five`. Likewise, this lets you say `move port five` but the previous one required you to say `move win port five` or `move window port five`. Is this intentional?

File: apps/i3wm/i3wm.talon (hunk 1 above)
Comment (EdJoJob) [reply apps/i3wm/i3wm.talon:66]: `port flip | flipper` moves an entire workspace (think virtual desktop) between outputs (monitors)
this is moving a window between workspaces

**Comment by rntz** [comment]
@splondike if you think this PR is useful, could you make the changes you think it ought to have? then we can merge it.

**Comment by splondike** [comment]
Once we have #1027 merged I'll convert this PR to do the following:
* Deprecate the 'murder' command.
* Remove explicit 'port ten'.
* Add 'shuffle|move flipper' command.

I'll leave out the other changes that adjust the grammar since they don't have a particular reason that I can see at the moment.

**Comment by splondike** [comment]
Since I'm pretty much reworking all of this I just made a new PR: #1081 . Closing this one out.

**Pull request closed without merging**